internal/service/federatedsettingsidentityprovider/resource_federated_settings_identity_provider.go

```diff
@@ -346,7 +338,7 @@ func resourceMongoDBAtlasFederatedSettingsIdentityProviderDelete(ctx context.Con
 }
 
 func resourceMongoDBAtlasFederatedSettingsIdentityProviderImportState(ctx context.Context, d *schema.ResourceData, meta any) ([]*schema.ResourceData, error) {
-	conn20231001002 := meta.(*config.MongoDBClient).Atlas20231001002
+	connV2 := meta.(*config.MongoDBClient).AtlasV2
```

Comment:
isn't older sdk needed anymore in federated resource or still using in some part?

Reply:
yes it's needed, but in this case it's used inside the method `oldSDKImport` [below](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1842/files#diff-6a7f7172746713f8b0bae40618f06b2903eec6958d6b1ff8f1c9a562d2ca10f9R349)